src/utils/text-mapper.ts

```diff
@@ -0,0 +1,144 @@
+            return 'Ikke vurdert';
+        case 'OPPFI':
+            return 'Helserelatert arbeidsrettet oppfølging i NAV';
+        case 'VURDI':
+            return 'Sykmeldt, oppfølging på arbeidsplassen';
+        case 'VURDU':
+            return 'Sykmeldt uten arbeidsgiver';
+        case 'BKART':
+            return 'Behov for arbeidsevnevurdering';
+        case 'KAP11':
+            return 'Rettigheter etter Ftrl. Kapittel 11';
+        default:
+            return EMDASH;
+    }
+}
+
+export function mapHovedmalTilTekst(hovedmal: OrNothing<Hovedmal | ArenaHovedmalKode>): string {
+    switch (hovedmal) {
+        case Hovedmal.BEHOLDE_ARBEID:
+        case 'BEHOLDEA':
+            return 'Beholde arbeid';
+        case Hovedmal.OKE_DELTAKELSE:
+        case 'OKEDELT':
+            return 'Øke deltakelse eller mål om arbeid';
+        case Hovedmal.SKAFFE_ARBEID:
+        case 'SKAFFEA':
+            return 'Skaffe arbeid';
+        default:
+            return EMDASH;
+    }
+}
+
+export function mapInnsatsgruppeTilTekst(innsatsgruppe: OrNothing<Innsatsgruppe | ArenaServicegruppeKode>): string {
+    switch (innsatsgruppe) {
+        case Innsatsgruppe.STANDARD_INNSATS:
+        case 'IKVAL':
+            return 'Standardinnsats';
+        case Innsatsgruppe.SITUASJONSBESTEMT_INNSATS:
+        case 'BFORM':
+            return 'Situasjonsbestemt innsats';
+        case Innsatsgruppe.SPESIELT_TILPASSET_INNSATS:
+        case 'BATT':
+            return 'Spesielt tilpasset innsats';
+        case Innsatsgruppe.GRADERT_VARIG_TILPASSET_INNSATS:
+            return 'Gradert varig tilpasset innsats';
+        case Innsatsgruppe.VARIG_TILPASSET_INNSATS:
+        case 'VARIG':
+            return 'Varig tilpasset innsats';
+        default:
+            return EMDASH;
+    }
+}
+export function hentOppfolgingsEnhetTekst(
+    oppfolgingsstatus: OppfolgingsstatusData | null | undefined
+): StringOrNothing {
+    const enhetId = oppfolgingsstatus?.oppfolgingsenhet?.enhetId;
+    const navn = oppfolgingsstatus?.oppfolgingsenhet?.navn;
+
+    if (!enhetId && !navn) {
+        return EMDASH;
+    }
+    if (!enhetId) {
+        return `${navn}`;
+    }
+    if (!navn) {
+        return `${enhetId}`;
+    }
+    return `${enhetId} ${navn}`;
+}
+
+export function hentGeografiskEnhetTekst(
+    personalia: PersonaliaV2Info | null | undefined
+): StringOrNothing {
+    const enhetsnummer = personalia?.geografiskEnhet?.enhetsnummer;
+    const navn = personalia?.geografiskEnhet?.navn;
+
+    if (!enhetsnummer && !navn) {
+        return EMDASH;
+    }
+    if (!enhetsnummer) {
+        return `${navn}`;
+    }
+    if (!navn) {
+        return `${enhetsnummer}`;
+    }
+    return `${enhetsnummer} ${navn}`;
+}
+
+export function hentVeilederTekst(veileder: VeilederData | null | undefined): StringOrNothing {
+    if (!veileder) {
+        return null;
+    }
+    if (!veileder.navn && !veileder.ident) {
+        return EMDASH;
+    }
```

Comment:
Tror dette kan skrives om til:

```suggestion
    if (!veileder?.navn && !veileder?.ident) {
        return EMDASH;
    }
```